src/ensembl/src/header/launchbar/Launchbar.tsx

```diff
@@ -81,7 +81,7 @@ const Launchbar = (props: LaunchbarProps) => {
           <div className={styles.category}>
             <LaunchbarButton
               app="help-docs"
-              description="Help & documentation"
+              description="Help &amp; documentation"
```

Comment:
But it works fine even with a regular ampersand (`&`). Why change to the html entity?

Reply:
Isn't it better and safer to use html entities?

Reply:
I can't imagine why. The browsers that we are targeting should be capable of properly displaying all unicode characters, the ampersand included.

Reply:
Ok. Hi @imransl This change was as per your request. Are you happy to revert it?

Reply:
Seems like `react-dom` does escape special characters before rendering them: https://medium.com/@arunrajeevan/how-react-helps-in-mitigating-cross-site-scripting-xss-attack-2f724c9e9242. So I wouldn't mind it being reverted, though I don't see any harm in leaving it like that either.

Reply:
ok. in that case I also vote for &.